src/awsexplorer/activation.ts

```diff
@@ -162,6 +163,13 @@ async function registerAwsExplorerCommands(
         )
     )
 
+    context.subscriptions.push(
+        vscode.commands.registerCommand(
+            'aws.importLambda',
+            async (node: LambdaFunctionNode) => await importLambdaCommand(node)
```

Comment:
Is there telemetry for this command/operation?

Reply:
Not right now, will add.